### PR TITLE
Create metrics-store-collected-metrics.html.md

### PR DESCRIPTION
**Pull request by sradco**
@JohnMarksRH  Please review this PR.

### CODE REVIEW

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  Amount of entropy available to the system.
+ 
+* [swap](https://collectd.org/wiki/index.php/Plugin:Swap)
+
+  Records the free, cached and used memory used by the swap and swap in/out.
+  
+* [df](https://collectd.org/wiki/index.php/Plugin:DF)
+
+  Collects information such as the free, reserved and used space of your files systems (including hard drive)
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
"Collects file system (incl. the hard drive) usage information such as free, reserved, and used space."

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  
+* [interface](https://collectd.org/wiki/index.php/Plugin:Interface)
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest "Interface"
And to remove bullet

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  Collects network trafic download and upload. It record the number of octets, packets and errors for transmission and reception on all network interface.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
Collects network traffic download and upload data, including throughput (octets per second), packets, and transmission errors..
Note "traffic"
This sentence requires further clarification. Let's discuss

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+* [aggregation](https://collectd.org/wiki/index.php/Plugin:Aggregation)
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest
"Aggregation"

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+  Aggregate multiple values into a single value using one or several consolidation functions, e.g. summation and average.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
Aggregates multiple values into a single value, using one or several consolidation functions, e.g. summation and average.

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  Currently it is used for calculating the average CPU utilization over all cores of each host.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
"It is currently used for calculating the average CPU utilization across all cores of each host."

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+* [processes](https://collectd.org/wiki/index.php/Plugin:Processes)
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
"Processes"
and bullet issue

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  
+  Process counts: Track the number of running, blocked, sleeping, paging, stopped and zombies process.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Number of running what?
Number of blocked what?
Requires clarification - let's talk

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+* [postgresql](https://collectd.org/wiki/index.php/Plugin:PostgreSQL) - Relevant only for oVirt engine
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest "PostgreSQL"
bullet issue

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+  PostgreSQL database statistics: custom_deadlocks, table_states, disk_io, disk_usage.
+
+
+## Logs collection
+oVirt logs are collected by fluentd, from the oVirt engine and hypervisors.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
oVirt logs are collected from the oVirt engine and hypervisors, by fluentd.

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+The data is then transformed, enriched and sent to the remote metrics store.
+
+Currently logs collected from engine machine:
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Do you mean?
"Currently, the logs collected from the engine machine include:"

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+### oVirt Metrics - collectd loaded plugins list
+
+* [disk](https://collectd.org/wiki/index.php/Plugin:Disk)
+  
+  Disk utilization: Sectors read/written, number of read/write actions,
+  average time an IO-operation took to complete.
+
+* [cpu](https://collectd.org/wiki/index.php/Plugin:CPU)
+
+  CPU time spent in the system, user, nice, idle, wait, interrupt, softirq and steal.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: It supposed to be nice ...

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
+## Statistics collection
+oVirt statistics are collected by collectd, from the oVirt engine and hypervisors and passed to fluentd.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+The data is then transformed, enriched and sent to the remote metrics store.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+### oVirt Metrics - collectd loaded plugins list
+
+* [disk](https://collectd.org/wiki/index.php/Plugin:Disk)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+  
+  Disk utilization: Sectors read/written, number of read/write actions,
+  average time an IO-operation took to complete.
+
+* [cpu](https://collectd.org/wiki/index.php/Plugin:CPU)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+
+  CPU time spent in the system, user, nice, idle, wait, interrupt, softirq and steal.
+      
+* [memory](https://collectd.org/wiki/index.php/Plugin:Memory)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+
+  Collects information about the used, buffered, cached and free RAM memory
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+
+* [load](https://collectd.org/wiki/index.php/Plugin:Load)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+
+  The number of runnable tasks in the run-queue, average over the last 1, 5 and 15 minutes.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+
+* [virt](https://collectd.org/wiki/index.php/Plugin:virt) - Relevant only for oVirt hypervisors
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+
+  CPU, memory, disk and network I/O statistics from virtual machines.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+ 
+* [nfs](https://collectd.org/wiki/index.php/Plugin:NFS)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+
+  NFS usage: A count of the number of procedure calls for each procedure, grouped by version
+  and whether the system runs as server or client. NFS version 4 is not supported for now.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+
+* [entropy](https://collectd.org/wiki/index.php/Plugin:Entropy)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 26]
```diff
@@ -0,0 +1,70 @@
...
+
+  Amount of entropy available to the system.
+ 
+* [swap](https://collectd.org/wiki/index.php/Plugin:Swap)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 27]
```diff
@@ -0,0 +1,70 @@
...
+
+  Records the free, cached and used memory used by the swap and swap in/out.
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 28]
```diff
@@ -0,0 +1,70 @@
...
+  
+* [df](https://collectd.org/wiki/index.php/Plugin:DF)
```
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 1 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 2 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 4 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 6 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done.

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 7 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 9 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 11 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Yes. Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md (hunk 10 above)
Comment (sradco) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Done

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 29]
```diff
@@ -0,0 +1,69 @@
...
+
+[Aggregation](https://collectd.org/wiki/index.php/Plugin:Aggregation)
+
+  Aggregates multiple values into a single value, using one or several consolidation functions, e.g. summation and average.
+  
+[Processes](https://collectd.org/wiki/index.php/Plugin:Processes)
+  
+  Collects the number of processes, grouped by their state (incl. running, blocked, sleeping, paging, stopped and zombies).
+
+[PostgreSQL](https://collectd.org/wiki/index.php/Plugin:PostgreSQL) - Relevant only for oVirt engine.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
"Applies only to oVirt engine."

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 30]
```diff
@@ -0,0 +1,69 @@
...
+
+  PostgreSQL database statistics: custom_deadlocks, table_states, disk_io, disk_usage.
+
+
+## Logs collection
+oVirt logs are collected from the oVirt engine and hypervisors, by fluentd.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Suggest:
fluentd collects oVirt logs from the oVirt engine and hypervisors.

File: source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md [hunk 31]
```diff
@@ -0,0 +1,69 @@
...
+
+  Records the free, cached, and used memory utilized by swap and swap in/out.
+  
+[DF](https://collectd.org/wiki/index.php/Plugin:DF)
+
+  Collects file system (incl. the hard drive) usage information such as free, reserved, and used space.
+  
+[Interface](https://collectd.org/wiki/index.php/Plugin:Interface)
+
+  Collects network traffic download and upload, including the number of octets, packets, and errors transmitted and received, for each network interface.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store-collected-metrics.html.md]: Collects network traffic download and upload (data OR statistics), including the number of octets, packets, and errors transmitted and received, for each network interface.

File: source/develop/release-management/features/engine/metrics-store.html.md [hunk 32]
```diff
@@ -52,7 +52,7 @@ Please follow one of these options. Adding SSO is optional:
 Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
-  * Openshit portal - <https://openshift.{hostname}>
+  * Openshift portal - <https://openshift.{hostname}>
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/engine/metrics-store.html.md]: Should be "OpenShift"  (capital S)

**Pull request merged**